### ISSUE TITLE
Can't get sample project working with instructions

### ISSUE THREAD

**Issue by berinle** [gradle/gradle-script-kotlin#4]
Its failing at the step to IDEA config for auto completion and what not

i.e. `./gradlew patchIdeaConfig`

Seems like it expects hello-world.iml to be right under modules directory, but it is under a subdirectory -597800014

Any insight will be much appreciated.

Error:

    $ ./gradlew patchIdeaConfig
    :patchIdeaConfig FAILED
    
    FAILURE: Build failed with an exception.
    
    * What went wrong:
    Execution failed for task ':patchIdeaConfig'.
    > java.io.FileNotFoundException: /private/tmp/gradle-script-kotlin/samples/hello-world/.idea/modules/hello-world.iml (No such file or directory)
    
    * Try:
    Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
    
    BUILD FAILED

Contents of .idea directory

    $ ll -R .idea
    total 184
    -rw-r--r--  1 berinle  wheel   870B May 18 11:00 compiler.xml
    drwxr-xr-x  3 berinle  wheel   102B May 18 11:00 copyright
    -rw-r--r--  1 berinle  wheel   159B May 18 11:00 encodings.xml
    -rw-r--r--  1 berinle  wheel    13K May 18 11:00 findbugs-idea.xml
    -rw-r--r--  1 berinle  wheel   684B May 18 11:00 gradle.xml
    drwxr-xr-x  5 berinle  wheel   170B May 18 11:02 libraries
    -rw-r--r--  1 berinle  wheel   1.4K May 18 11:00 misc.xml
    drwxr-xr-x  3 berinle  wheel   102B May 18 11:00 modules
    -rw-r--r--  1 berinle  wheel   682B May 18 11:00 modules.xml
    -rw-r--r--  1 berinle  wheel    55K May 18 11:02 workspace.xml
    
    .idea/copyright:
    total 8
    -rw-r--r--  1 berinle  wheel    74B May 18 11:00 profiles_settings.xml
    
    .idea/libraries:
    total 64
    -rw-r--r--  1 berinle  wheel   481B May 18 11:00 Gradle__junit_junit_4_12.xml
    -rw-r--r--  1 berinle  wheel   537B May 18 11:00 Gradle__org_hamcrest_hamcrest_core_1_3.xml
    -rw-r--r--  1 berinle  wheel    20K May 18 11:02 gradle_ide_support.xml
    
    .idea/modules:
    total 0
    drwxr-xr-x  5 berinle  wheel   170B May 18 11:00 -597800014
    
    .idea/modules/-597800014:
    total 96
    -rw-r--r--  1 berinle  wheel    12K May 18 11:00 hello-world.iml
    -rw-r--r--  1 berinle  wheel    12K May 18 11:00 hello-world_main.iml
    -rw-r--r--  1 berinle  wheel    13K May 18 11:00 hello-world_test.iml

**Set up:**
IntelliJ IDEA 2016.1.2
Build #IU-145.972, built on May 14, 2016
JRE: 1.8.0_76-release-b162 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

**Kotlin plugin**: https://teamcity.jetbrains.com/repository/download/bt345/766796:id/kotlin-plugin-1.1.0-dev-372.zip

**Comment by cbeams** [comment]
@berinle this is just a hunch, but what happens if you try to follow the instructions somewhere other than within `/tmp`?

**Comment by berinle** [comment]
I'll give that a spin now and report back

**Comment by berinle** [comment]
Same issue. This time, the modules directory is empty

    $ ./gradlew patchIdeaConfig
    :patchIdeaConfig FAILED
    
    FAILURE: Build failed with an exception.
    
    * What went wrong:
    Execution failed for task ':patchIdeaConfig'.
    > java.io.FileNotFoundException: /Users/berinle/workspace/study/gradle-script-kotlin/samples/hello-world/.idea/modules/hello-world.iml (No such file or directory)
    
    * Try:
    Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
    
    BUILD FAILED
    
    Total time: 5.297 secs


Listing contents of .idea

    $ ll -R .idea
    total 80
    -rw-r--r--  1 berinle  staff   695B May 18 12:09 compiler.xml
    drwxr-xr-x  3 berinle  staff   102B May 18 12:09 copyright
    -rw-r--r--  1 berinle  staff   159B May 18 12:09 encodings.xml
    -rw-r--r--  1 berinle  staff    13K May 18 12:09 findbugs-idea.xml
    drwxr-xr-x  3 berinle  staff   102B May 18 12:09 libraries
    -rw-r--r--  1 berinle  staff   5.9K May 18 12:09 misc.xml
    drwxr-xr-x  2 berinle  staff    68B May 18 12:09 modules
    -rw-r--r--  1 berinle  staff   6.2K May 18 12:09 workspace.xml
    
    .idea/copyright:
    total 8
    -rw-r--r--  1 berinle  staff    74B May 18 12:09 profiles_settings.xml
    
    .idea/libraries:
    total 48
    -rw-r--r--  1 berinle  staff    20K May 18 12:09 gradle_ide_support.xml
    
    .idea/modules:

**Comment by cbeams** [comment]
As a sanity check, I've just run through everything myself again. Here's what I did:
```
/Work/gradle/gradle-script-kotlin[master]
$ git log --oneline -1
9673e45 (HEAD -> master, origin/master) Return to 1.0.0-SNAPSHOT following 1.0.0-M1 release

/Work/gradle/gradle-script-kotlin[master]
$ # close all projects in IDEA and quit IDEA

/Work/gradle/gradle-script-kotlin[master]
$ git clean -dfx # blow away everything

/Work/gradle/gradle-script-kotlin[master]
$ cd samples/hello-world

$ ./gradlew --version

------------------------------------------------------------
Gradle 3.0-snapshot-1
------------------------------------------------------------

Build time:   null
Revision:     HEAD

Groovy:       2.4.4
Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
JVM:          1.8.0_60 (Oracle Corporation 25.60-b23)
OS:           Mac OS X 10.11.3 x86_64

/Work/gradle/gradle-script-kotlin/samples/hello-world[master]
$ ./gradlew generateKtsConfig
:generateKtsConfig

BUILD SUCCESSFUL

Total time: 2.075 secs

/Work/gradle/gradle-script-kotlin/samples/hello-world[master]
$ # open the project in IDEA as per samples/README

/Work/gradle/gradle-script-kotlin/samples/hello-world[master]
$ ./gradlew patchIdeaConfig
:patchIdeaConfig

BUILD SUCCESSFUL

Total time: 1.593 secs

/Work/gradle/gradle-script-kotlin/samples/hello-world[master]
$ # explore the project in IDEA, see that everything works as expected

/Work/gradle/gradle-script-kotlin/samples/hello-world[master]
$ tree .idea/modules
.idea/modules
├── hello-world.iml
├── hello-world_main.iml
└── hello-world_test.iml
```

So I'm unable to reproduce the issue at this point. Could you verify once more by following my steps above and seeing where things diverge? Thanks.

**Comment by cbeams** [comment]
Strange. Any chance you could try this on a different machine somewhere? We're seeing these instructions work consistently for us across a variety of local machines and CI boxes. At this point it looks like it's something specific to your setup that we're not yet seeing.

**Comment by berinle** [comment]
@cbeams thanks for your prompt response. I just followed your instructions step by step, and it works! I have repeated it several times again just to be sure it wasn't a fluke.

I really can't put my finger on what I might have been doing differently. Maybe something went wrong when importing the project initially, that caused the right files not to be generated in the `modules` directory. 

In any case, I'm all good now.

Apologies for the false alarm.

Cheers
